pyproject.toml

```diff
@@ -15,8 +15,8 @@ dependencies = [
     "aiohttp",
     "aiokafka",
     "aiozmq",
-    "apischema==0.16.1",
     "immutables",
+    "pydantic>=2.0",
```

Comment:
Prevents issues with importing pydantic.v1

Reply:
Find out which version of pydantic we're using and set directly?